dlib/cmake_utils/cmake_find_blas.txt

```diff
@@ -58,21 +62,23 @@ if (UNIX)
     check_type_size( "void*" SIZE_OF_VOID_PTR)
 
     if (SIZE_OF_VOID_PTR EQUAL 8)
-        set( mkl_search_path
```

Comment:
Why not just add $ENV{MKL_PATH} to the list of paths searched?  That would be much clearer, no if statements.  I'm not sure these statements are even right.  Above, mkl_search_path is set.  Do these if statements really fall through correctly if MKL_PATH is not set to something?